packages/kotti-ui/source/kotti-inline-edit-2/KtInlineEdit2.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+	<div :class="objectClass">
+		<label class="form-label" v-text="label" />
+		<div v-if="isEditing">
+			<component
+				:is="inputComponent"
+				v-bind="$attrs"
```

Comment:
also not in favor of this `v-bind:"$attrs"`
> if all we need is `placeholder`?

Reply:
why ?

Reply:
I'm guessing we also need labels